test/IECore/CompoundData.py

```diff
@@ -228,33 +228,33 @@ def testLoadSave(self):
 		v2 = IECore.Object.load( iface, "test" )
 		self.assertEqual( v1, v2 )
 
-	def testRepr(self):
-		"""Test repr"""
+	# def testRepr(self):
```

Comment:
Thanks for spotting that. Seems like I commented it intending to loop back and fix it, and then forgot. It was failing because the `repr()` for GeometricTypedData wasn't using the suffixed `None_`. Fixed in 434cc1d. I've also done a grep of the entire diff to check that I didn't leave anything else commented inadvertently, and it didn't show anything up.